shared/util/quit-helper.desktop.js

```diff
@@ -47,7 +56,8 @@ function _executeActionsFromRenderer (actions: Array<Action>) {
   ipcRenderer.send('executeActions', actions)
 }
 
-export function executeActions (actions: Array<Action>) {
+export function executeActionsForContext (context: Context) {
```

Comment:
I'm okay having a helper that does `executeActionsForContext` but I think we should keep `executeActions` since it's a smaller piece that composes better.

Then executeActionsForContext is just a composition of those two functions.